docs/dsp-tools-xmlupload.md

```diff
@@ -437,6 +465,7 @@ Attributes:
 - `permissions`: ID or a permission set (optional, but if omitted, very restricted default permissions apply)
```

Comment:
@irinaschubert Here I am not certain what "ID or a permission set" means. I know only one possibility of adding a permission, but here it sounds if it were 2 possibilities...